src/components/ValuesScreen.tsx

```diff
@@ -1,28 +1,299 @@
+  return (
+    
+      <Button theme={{ roundness: 30 }} contentStyle={{width: 240, height: 70}} compact={true} mode="outlined" onPress={props.categoryButton}>
+        <Text>{props.name}</Text>
+      </Button>
+  )
+}
+
+//View when adding a new topic
+const TopicTextInputView = ({route, navigation}: any) => {
+  const [values, modifyValues] = Storage.useValues();
+  const [text, setText] = React.useState('');
+  const [people, modifyPeople] = Storage.usePeople();
+  const { title, navigateBack, categoryString } = route.params;
   return (
-    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
-      <Text>Values</Text>
+    <View style={{flex: 1}}>
+      <View style={{flex: 0.3, justifyContent: 'center', alignItems: 'center'}}>
+        <Title style={{fontSize: 30}}>{title}</Title>
+      </View>
+      <View style={{flex: 0.2, justifyContent: 'center'}}>
+      <TextInput
+      value={text}
+      onChangeText={setText}
+      mode={"outlined"} 
+      style={{flex: 0.5, paddingHorizontal: 50, justifyContent: 'flex-start', fontSize: 20}}
+      placeholder={"Skriv här"}
+      multiline={true}
+    />
+      </View>
+      <View style={{ flex: 0.3, paddingBottom: 60, flexDirection: 'row', alignItems: 'flex-end', justifyContent: 'space-evenly'}}>
+        <RoundButton icon="close" size={40} onPress={() => {
+          navigation.navigate({
+            name: navigateBack
+          })
+        }} />
+        <RoundButton icon="check" size={40} onPress={() =>{
+          navigation.navigate({
+            name: navigateBack,
+          },);
+          if(categoryString != 'people'){
+            modifyValues.addTopic(categoryString, text);
+          }
+          else{
+            modifyPeople.add(text);
+          }
+
+        } } />  
+      </View>
+     </View>
+  )
+}
+
+//View for each topic
+const TopicView = ({route, navigation}: any) => {
+  const { title, navigateBack, categoryString} = route.params;
+  const [values, modifyValues] = Storage.useValues();
+  const [people, modifyPeople] = Storage.usePeople();
+  let category: any;
+  let buttonText: any;
+  let content: any;
+  
+  //Create buttons to each topic whenever we add or delete a topic
+  useEffect(() => {
+    buttons
+  }, [values])
+
+  //Create text boxes to people whenever we add or delete a people
+  useEffect(() => {
+    textBoxes
+  }, [people])
+
+  //To find the right category
+  switch(categoryString){
+    case "relations":
+      category = values.relations;
+      break;
+    case "work":
+      category = values.work;
+      break;
+    case "health":
+      category = values.health;
+      break;
+    case "enjoyment":
+      category = values.enjoyment;
+      break;
+    case "responsibilities":
+      category = values.responsibilities;
+      break;
+    case "people":
+      category = people;
+      break;
+    //Category needs to be set to something
+    default:
+      category = values.relations;
+
+  }
```

Comment:
```suggestion
  category = values[categoryString];
```
You can't just do this?